### PR TITLE
Pashto language localization

### PR DESCRIPTION
**Pull request by isshort**

### CODE REVIEW

**Comment by isshort** [comment]
This branch is linked to #310

**Comment by larryaasen** [comment]
@isshort Thanks for your PR. The 'ps' language name should be listed after 'pl' and before 'ru' to keep them alphabetized in English. Can you make that change?

**Review by larryaasen — CHANGES_REQUESTED** [review]
@isshort Thanks for your PR. The 'ps' language name should be listed after 'pl' and before 'ru' to keep them alphabetized in English. Can you make that change?

File: lib/src/upgrade_messages.dart [hunk 1]
```diff
@@ -135,6 +135,7 @@ class UpgraderMessages {
         message =
             'نسخه‌ی جدیدی از {{appName}} موجود است! نسخه‌ی {{currentAppStoreVersion}} در دسترس است ولی شما همچنان از نسخه‌ی {{currentInstalledVersion}} استفاده می‌کنید.';
         break;
+
```
Comment (larryaasen) [inline lib/src/upgrade_messages.dart:138]:
```suggestion

```

**Comment by larryaasen** [comment]
This change was published in version 11.2.0.
https://pub.dev/packages/upgrader/versions/11.2.0

**Pull request merged**